### PR TITLE
Settings zsofi w. sharedpreferences

### PR DESCRIPTION
**Pull request by zsofinnemeth**

### CODE REVIEW

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/AppCompatPreferenceActivity.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.greenfox.fuchsit.zerdareader.activity;
+
+/**
+ * Created by false on 2017. 02. 07..
+ */
+
+public class AppCompatPreferenceActivity {
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/AppCompatPreferenceActivity.java]: @zsofinnemeth why is this here?

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/AppCompatPreferenceActivity.java (hunk 1 above)
Comment (zsofinnemeth) [reply app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/AppCompatPreferenceActivity.java]: @adamgyulavari sorry, this is a forgotten 'leftover' from one of the solutions I was trying to use. I deleted it and pushed again

**Review by adamgyulavari — APPROVED** [review]
